### PR TITLE
V11-alpha

### PR DESCRIPTION
**Pull request by iondrimba**
**CHANGELOG** :memo:

# WIP

These following components/settings were updated:

* Heading
* Buttons
* Card
* Dialog
* Form
* List
* Pagination 
* Thumbnail
* Media / Media Box
* Borders, Colors, Shadows, Spaces
* Documentation layout

### CODE REVIEW

File: src/scss/components/_card.scss [hunk 1]
```diff
@@ -26,10 +26,35 @@
 .card {
   background-color: get-color(air);
   border-radius: $borderRadius;
-  box-shadow: 0 4px 8px 0 get-color((color: blackNeutral, opacity: .12));
+  box-shadow: 0 2px 4px 0 get-color((color: blackNeutral, opacity: .12));
```
Comment (eduardojmatos) [inline src/scss/components/_card.scss]: @iondrimba can we use `get-shadow(medium)` here? 😄

File: src/scss/tools/mixins/_generate_button_types.scss [hunk 2]
```diff
@@ -23,15 +37,26 @@
...
+    }
+
+    @if map-has-key($buttonTypeConfig, borderTopColor) {
+      border-top-color: get-color(map-get($buttonTypeConfig, borderTopColor));
+    }
+
     @if map-has-key($buttonTypeConfig, shadow) {
-      box-shadow: 6px 6px 20px 0 get-color(map-get($buttonTypeConfig, shadow));
+      $shadow: map-deep-get($buttonTypeConfig, shadow, kind);
+      box-shadow: map-get($shadows, $shadow) get-color((color: blackNeutral, key: darker, opacity: .2));
```
Comment (eduardojmatos) [inline src/scss/tools/mixins/_generate_button_types.scss]: @iondrimba did you try `get-shadow($shadow)` here or do we have some troubles with functions and variables with string as a key?

File: src/scss/components/_card.scss (hunk 1 above)
Comment (iondrimba) [inline src/scss/components/_card.scss]: @eduardojmatos Yes, we can 👍

File: src/scss/tools/mixins/_generate_button_types.scss (hunk 2 above)
Comment (iondrimba) [inline src/scss/tools/mixins/_generate_button_types.scss]: @eduardojmatos It worked, :1st_place_medal: 🎉

File: src/scss/components/_card.scss (hunk 1 above)
Comment (eduardojmatos) [inline src/scss/components/_card.scss]: image

**Pull request merged**